test/src/test/java/org/corfudb/infrastructure/LayoutBuilderTest.java

```diff
@@ -92,6 +91,7 @@ public void checkRemovalOfNodes() throws LayoutModificationException {
         failedNodes.clear();
         failedNodes.add(allNodes.get(0));
         failedNodes.add(allNodes.get(2));
+        failedNodes.add(allNodes.get(3));
```

Comment:
Travis is failing due to magic number 3.

Reply:
Addressed. Thank.s